dev-docs/bidders/viewdeosDX.md

```diff
@@ -14,8 +14,30 @@ gdpr_supported: true
 | Name  | Scope    | Description                     | Example  | Type      |
 |-------|----------|---------------------------------|----------|-----------|
 | `aid` | required | The source ID from member zone | `350975` | `integer` |
+| `outstream` | optional | Oustream player settings object | `{"audio_setting":"on","video_controls":"show"}` | `object` |
 
+#### Oustream settings object
 
+{: .table .table-bordered .table-striped }
+| Name  | Scope    | Description                     | Possible Values  | Type      |
```

Comment:
I don't think "Possible Values" is the right column header here. Would suggest changing to "Example" as in other parameter tables. Then move the universe of possible values to the description.